appinfo.go

```diff
@@ -0,0 +1,61 @@
+package harmony
+
+import (
+	"encoding/json"
+	"context"
+	"net/http"
+	"github.com/skwair/harmony/internal/endpoint"
```

Comment:
Imports are not properly formatted with `goimports` (though golangci-lint was doing this check by default but it turns out it doesn't :cry:, I will enable this and disable the e2e tests since they can't be run for PRs made from forks)